cmd/textiled/main.go

```diff
@@ -240,6 +245,7 @@ var rootCmd = &cobra.Command{
 			EmailFrom:            emailFrom,
 			EmailDomain:          emailDomain,
 			EmailApiKey:          emailApiKey,
+			ThreadsInternalToken: uuid.New().String(),
```

Comment:
I see this token spreading deep into things (but maybe not, maybe the other ones are actually user/app user tokens now that I think about it). But what is this one used for?

Reply:
This is used as a means to identify requests to the thread server _from textile itself_, like projects, teams, etc. Otherwise, those requests would return as unauthenticated. The interceptor simply lets all requests go through if they come with this authorization token, which is only ever visible to `textile` itself (not visible outside the box / process).